build/tools/cosmovisor.md

```diff
@@ -0,0 +1,55 @@
+# Cosmovisor
+
+_This doc is to be reviewed and updated_
+
+**cosmovisor** is a small process manager for any Cosmos SDK based application binaries that monitors the governance module for incoming **chain upgrade proposals**. If it sees a proposal that gets approved, cosmovisor can automatically download the new binary, stop the current binary, switch from the old binary to the new one, and finally restart the node with the new binary.
+
+For detailed information related to cosmovisor, refer here: [cosmos-sdk doc](https://docs.cosmos.network/main/tooling/cosmovisor).
+
+## Auto-Download feature
+
+If the system administrator plans the Persistence chain upgrade using Cosmovisor, generally the upgrade and current binaries need to be manually placed on the disk before the upgrade happens.  
+
+Once the cosmovisor process is started, below folder structure is automatically created in the chain directory:-  
+
+![alt text for screen readers](cosmo-visor-folder-layout.jpg)
+
+Before the chain upgrade, target version chain binary needs to be added under cosmovisor/upgrades/upgrade-name/bin directory manually.
+However, for people who don't need such control and want an automated setup to download binaries on all the validating nodes, there is an **auto-download** option.
+
+### Steps to enable the Auto-Download feature
+1. Ensure to set environment variable **DAEMON_ALLOW_DOWNLOAD_BINARIES** to **true** before starting the cosmovisor process
```

Comment:
Is this in the .service file which runs the node (using cosmovisor)? Can we clarify?

Reply:
For these steps, I have added reference to existing cosmovisor steps on the "Join a Mainnet" and "Join a Testnet" page itself.